templates/ts/notification-restify/package.json.tpl

```diff
@@ -10,6 +10,8 @@
     "main": "./lib/index.js",
     "scripts": {
         "dev:teamsfx": "env-cmd --silent -f .localConfigs npm run dev",
+        "dev:teamsfx:testtool": "env-cmd --silent -f .localConfigs.testTool npm run dev",
+        "dev:teamsfx:launch-testtool": "env-cmd --silent -f env/.env.testtool teamsapptester start",
```

Comment:
how about consolidate the file name for `.localConfigs.testTool` and `.env.testtool`? maybe .env.testtool => .env.testTool?

Reply:
looks like a legacy issue, resolved.